src/main/java/org/sasanlabs/internal/utility/JSONSerializationUtils.java

```diff
@@ -17,6 +17,8 @@ public class JSONSerializationUtils {
 
     private static final ObjectMapper MAPPER = new ObjectMapper();
 
+    private static JSONSerializationUtils() {}
```

Comment:
static constructor?

Reply:
my bad, that's an oversight. I'll address it in next commit